packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -778,10 +798,7 @@ const DataTable = ({
     // Refs
     muiTableProps: {
       ref: tableContentRef,
-      className:
-        !shouldDisplayRowActions && hasColumnResizing
-          ? "ods-hide-spacer-column"
-          : "",
+      className: !shouldDisplayRowActions && hasColumnResizing ? "" : "",
```

Comment:
@jordankoschei-okta We are never setting any className here. Might as well just remove this prop or set it as empty string.

Reply:
Also, does removing this reintroduce weird last column issues we've seen in the past?